frontend/public/components/filter-toolbar.tsx

```diff
@@ -100,8 +100,11 @@ const FilterToolbar_: React.FC<FilterToolbarProps & RouteComponentProps> = (prop
   const [inputText, setInputText] = React.useState('');
   const [filterType, setFilterType] = React.useState(FilterType.NAME);
   const [isOpen, setOpen] = React.useState(false);
-  const placeholder =
-    filterPlaceholder || (FilterType.NAME === filterType ? 'Search by name...' : 'app=frontend');
+  const [placeholder, setPlaceholder] = React.useState(
+    FilterType.NAME === filterType
+      ? `Search by ${kind !== undefined ? kind.toLowerCase() : ''} name...`
```

Comment:
I would just lowercase `kind`. It is awkward for longer names since the word boundaries are no longer apparent.

Reply:
The good reasons for using this approach and not just using nameFIlterPlaceholder  :- 

1. We won't  have to hardcode the resource ( Node ) and the type of filter (Name, Label)
1. filter-toolbar being a generic component can accommodate multiple types of filter in the future. This could handle those scenarios too.
1. We just can pass the type of resource by kind prop and let the filter automatically handle any resource types.

Reply:
> I would just lowercase kind. It is awkward for longer names since the word boundaries are no longer apparent.

Sorry i didn't get this point. Could you explain a bit more ?

Reply:
You shouldn't just manipulate kind with `toLowercase`. Things like `SecurityContextContraint` become `securitycontextconstraint` which is hard to read. You would have to use the model display name.

Won't this change all the list pages instead of the one we want to change?

Reply:
Yes i understand your concern on readability in which case we may need to keep non lower cased name. 
> Won't this change all the list pages instead of the one we want to change?

Yes that was the initial idea that if we are not passing the kind prop, it will just show 'Search by name' instead of 'Search by {kind} name'